### ISSUE TITLE
Configuration variables are not defined as defcustom

### ISSUE THREAD

**Issue by doolio** [UndeadKernel/pacfiles-mode#6]
Firstly, thanks for this package. I was bitten by a recent change to some `PAM` config files and after resolving that issue I looked to see if I had inadvertenly neglected others in the past. Your package alerted me to several pacfiles I had neglected due to inexperience. So thank you.

The README lists a set of configuration variables which suggests these are customisable by the user which they are not. Should they be re-defined as `defcustom`s or clarified in the README?

Thanks for your time and this package.

**Comment by UndeadKernel** [comment]
Hey @doolio.
I personally don't use the `custom` emacs facility but I guess it's a good idea to support it. However, there is only one variable that is meant to be changed by the user: `pacfiles-updates-search-command`. Well, you can technically also want to change `pacfiles-merge-file-tmp-location`. In the coming days I will make those two variables work with the `customize` framework.

By the way, you are free to change any variable in your init.el file with something like:
```lisp
(with-eval-after-load "pacfiles-mode"
    (setq pacfiles-updates-search-command
        "find / -name '*.pacnew' -o -name '*.pacsave' 2>/dev/null"))
```

**Comment by doolio** [comment]
```
In the coming days I will make those two variables work with the customize framework.
```
Thanks, I might be unique in liking how the `custom` facility organises the various settings.

**Comment by UndeadKernel** [comment]
Solved in  8d06f64abc98c3f3338560c8d6eb47719e034069
